src/test/java/seedu/address/logic/commands/AliasCommandIntegrationTest.java

```diff
@@ -22,20 +22,39 @@ public void setUp() {
 
     @Test
     public void execute_createAliasUseAliasDeleteAlias_success() {
-        String alias = "TEST ALIAS";
-        String aliasTo = "help";
+        String alias1 = "TEST ALIAS";
+        String aliasTo1 = "help";
+        String aliasUsage1 = "TEST ALIAS";
+        String aliasUsageExpected1 = "help";
+        String alias2 = "TEST ADD";
+        String aliasTo2 = "add";
+        String aliasUsage2 = "TEST ADD n/John Smith p/91234567 e/john@smith.co.uk "
+                + "a/12 Downing St Westminster, London SW1A 2AD, UK";
+        String aliasUsageExpected2 = "add n/John Smith p/91234567 e/john@smith.co.uk "
+                + "a/12 Downing St Westminster, London SW1A 2AD, UK";
 
```

Comment:
Test Aliases could perhaps be set as a static variable in a separate TestUtil class actually. Something like TypicalAliases() could be useful for future extension of the alias feature.